packages/teleport-uidl-resolver/src/utils.ts

```diff
@@ -138,7 +138,9 @@ export const resolveElement = (element: UIDLElement, options: GeneratorOptions)
       if (attrValue.type === 'static' && typeof attrValue.content === 'string') {
         originalElement.attrs[attrKey].content = UIDLUtils.prefixAssetsPath(
           attrValue.content,
-          options.assets
+          originalElement.elementType === 'Link' || originalElement.elementType === 'a'
```

Comment:
This acts kinda of hack, let's come up with some solution. But ideally a link should me missing from assets mapping. So, we should never run into this issue 🤔

Reply:
The problem is not about the assets mapping (the link is not in the asset mapping), the issue is either in the fact that local links are accessing this prefixing function or in the prefixing function itself 
`if (!mappings[assetName]) {
    if (!identifier) {
      return [prefix, assetName].join('/')
    }
    return [prefix, identifier, assetName].join('/')
  }`
  
  The problem is that if we have an identifier (or a prefix or both) defined in the options.assets (which is necessary for assets) it will be appended to a local link even though it doesn't make sense to use assets related properties on it